public/index.htm

```diff
@@ -19,8 +19,7 @@
     />
     <div id="menuDialog" title="Configuration Settings">
         <form id="menuForm">
-            <div class="menuButton">update</div>
-            <div class="menuButton">load defaults</div>
```

Comment:
Was this "load defaults" button just a placeholder or is there logic somewhere which must also be removed?

Reply:
yes it was a dead placeholder, should not exist till it works.